pages/support.js

```diff
@@ -13,21 +16,17 @@ import Hero from '../src/Hero'
 import TrySection from '../src/TrySection'
 import Popover from '../src/Popover/Popover'
 
-const HeadInjector = () => (
-  <Head>
-    <title>Support | Machine Learning Version Control System</title>
-  </Head>
-)
-
-export default () => (
```

Comment:
is lint complaining about export default?

Reply:
It did complain that the export needs a name, yes. So I named it `const SupportPage` and moved the `export` statement to the bottom of the file.